Update information-barriers.md (#3750)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/information-barriers.md

Microsoft cloud services include powerful communication and collaboration capabilities. But suppose that you want to restrict communication and collaboration between two groups to avoid a conflict of interest from occurring in your organization. Or, perhaps you want to restrict communication and collaboration between certain people inside your organization in order to safeguard internal information. Microsoft 365 enables communication and collaboration across groups and organizations, so is there a way to restrict communication and collaboration  among specific groups of users when necessary? With information barriers, you can! 

> Information barriers is now supported in Microsoft Teams, SharePoint Online, and OneDrive for Business. Assuming your [subscription](#required-licenses-and-permissions) includes information barriers, a compliance administrator or information barriers administrator can define policies to allow or prevent communication and collaboaration between groups of users in Microsoft Teams, SharePoint Online and OneDrive. Information barrier policies can be used for situations like these:

Reviewer: ```suggestion
Information barriers are now supported in Microsoft Teams, SharePoint Online, and OneDrive for Business. Assuming your [subscription](#required-licenses-and-permissions) includes information barriers, a compliance administrator or information barriers administrator can define policies to allow or prevent communication and collaboration between groups of users in Microsoft Teams, SharePoint Online, and OneDrive. Information barrier policies can be used for situations like these:
```

- User in the day trader group should not communicate or share files with the marketing team
- Finance personnel working on confidential company information should not communicate or share files with certain groups within their organization
- An internal team with trade secret material should not call or chat online with people in certain groups within their organization
- A research team should only call or chat online with a product development team

> - A site for day trader group should not be shared or accessed by anyone outside the day trader group

Reviewer: ```suggestion
- A site for a day trader group should not be shared or accessed by anyone outside the day trader group
```

> [!IMPORTANT]

> > Information barriers ***only supports*** two way restrictions. One way restrictions, such as marketing can communicate and collaborate with day traders, but day traders cannot communicate and collaborate with marketing ***is not supported***.

Reviewer: ```suggestion
> Information barriers ***only support*** two-way restrictions. One-way restrictions, such as marketing can communicate and collaborate with day traders, but day traders cannot communicate and collaborate with marketing, *** are not supported***.
```

> For all of these example scenarios (and more), information barrier policies can be defined to prevent or allow communications and collaboration in Microsoft Teams, SharePoint Online and OneDrive. Such policies can prevent people from calling or chatting with those they shouldn't, or enable people to communicate only with specific groups in Microsoft Teams. With information barrier policies in effect, whenever users who are covered by those policies attempt to communicate and collaborate with others in Microsoft Teams, SharePoint Online or OneDrive checks are done to prevent (or allow) communication and collaboration (as defined by information barrier policies). To learn more about the user experience with information barriers, see [information barriers in Microsoft Teams](https://docs.microsoft.com/MicrosoftTeams/information-barriers-in-teams) and [information barriers in SharePoint Online](https://docs.microsoft.com/sharepoint/information-barriers)

Reviewer: ```suggestion
For all of these example scenarios (and more), information barrier policies can be defined to prevent or allow communications and collaboration in Microsoft Teams, SharePoint Online, and OneDrive. Such policies can prevent people from calling or chatting with those they shouldn't, or enable people to communicate only with specific groups in Microsoft Teams. With information barrier policies in effect, whenever users who are covered by those policies attempt to communicate and collaborate with others in Microsoft Teams, SharePoint Online or OneDrive checks are done to prevent (or allow) communication and collaboration (as defined by information barrier policies). To learn more about the user experience with information barriers, see [Information barriers in Microsoft Teams](https://docs.microsoft.com/MicrosoftTeams/information-barriers-in-teams) and [Use information barriers with SharePoint](https://docs.microsoft.com/sharepoint/information-barriers).
```

> [!IMPORTANT]
> Currently, information barriers do not apply to email communications. In addition, information barriers are independent from [compliance boundaries](set-up-compliance-boundaries.md).<p>Before you define and apply information barrier policies, make sure your organization does not have [Exchange address book policies](https://docs.microsoft.com/exchange/address-books/address-book-policies/address-book-policies) in effect. (Information barriers are based on address book policies.) 

## What happens with information barriers

When information barrier policies are in place, people who should not communicate or share files with other specific users won't be able to find, select, chat, or call those users. With information barriers, checks are in place to prevent unauthorized communication and collaboration. 

> Information barriers applies to Microsoft Teams (chats and channels), SharePoint Online and OneDrive. 

Reviewer: ```suggestion
Information barriers apply to Microsoft Teams (chats and channels), SharePoint Online, and OneDrive. 
```

> To learn more about the user experience with information barriers, see [information barriers in Microsoft Teams](https://docs.microsoft.com/MicrosoftTeams/information-barriers-in-teams).

Reviewer: ```suggestion
To learn more about the user experience with information barriers, see [Information barriers in Microsoft Teams](https://docs.microsoft.com/MicrosoftTeams/information-barriers-in-teams).
```

In SharePoint Online and OneDrive, information barrier policies determine and prevent the following kinds of unauthorized collaborations:

- Adding a member to a site

> - Accessing site or content by a user

Reviewer: ```suggestion
- Accessing a site or content by a user
```

> - Sharing site or content with another user

Reviewer: ```suggestion
- Sharing a site or content with another user
```

- Searching a site 

> To learn more about the user experience with information barriers, see [information barriers in SharePoint Online](https://docs.microsoft.com/sharepoint/information-barriers)

Reviewer: ```suggestion
To learn more about the user experience with information barriers, see [Use information barriers with SharePoint](https://docs.microsoft.com/sharepoint/information-barriers).
```
